blog/2024-10-25-object-notifications/index.mdx

```diff
@@ -0,0 +1,132 @@
+---
+slug: how-we-built-object-notifications
+title: How we built object notifications in Tigris
+description: >
+  A deep dive into the implementation of object notifications in Tigris.
```

Comment:
Tigris now supports object notifications! Object notifications are how you receive events every time something changes in a bucket. This post is a deep dive into the implementation of object notifications in Tigris.